src/main/resources/application.yml

```diff
@@ -62,3 +62,6 @@ hapi-fhir:
     uri: /Patient
   bundle:
     uri: /Bundle
+
+feature-flag:    
+  enforcePatientId: true
```

Comment:
Should we put this behind an environment variable?

Reply:
@chubert-sb We sure can!

Reply:
@chubert-sb Just pushed in the changes for environment variable. Please take a look again, thanks!